.sequelizerc

```diff
@@ -1,7 +1,7 @@
-import path from 'path';
-import '@babel/register';
+require('@babel/register');
+const path = require('path');
 
-export default {
+module.exports = {
```

Comment:
@shemaeric  same applies here, please stick to es6

Reply:
@aaronsekisambu With ES6 the file doesn't run so we decided to revert back to ES5 meanwhile we are looking for a solution to run it in ES6